src/DropdownMenu/DropdownMenu.js

```diff
@@ -9,17 +9,26 @@ export default function DropdownMenu({
   anchorEl,
   children,
   anchorOrigin,
+  anchorPosition,
   transformOrigin,
 }) {
+  const props = {
+    open,
+    onClose,
+    style: { ...style },
+    anchorOrigin,
+    transformOrigin,
+  };
+
+  if (anchorPosition) { // use absolute positioning to specified coordinates
+    props.anchorReference = 'anchorPosition';
+    props.anchorPosition = anchorPosition;
+  } else { // position relative to another component
+    props.anchorEl = anchorEl;
+  }
+
   return (
-    <Popover
-      open={open}
-      onClose={onClose}
-      style={{ ...style }}
-      anchorEl={anchorEl}
-      anchorOrigin={anchorOrigin}
-      transformOrigin={transformOrigin}
-    >
```

Comment:
I suggest adding the following instead of making a copy of the props object

anchorReference={anchorPosition ? 'anchorPosition' : null}
anchorPosition={anchorPosition || null}